ansible_collections/arista/avd/roles/dhcp_provisioner/README.md

```diff
@@ -103,6 +103,22 @@ spine:
       mac_address: '0c:1d:c0:1d:62:02'
 ```
 
+This module also use `cvp_instance_ip` / `cvp_instance_ips` knob to set correct registration URL. So in case you need to extend TerminAttr daemon, you should use an approach similar to the following code:
+
+```yaml
+cvp_instance_ips:
+  - 10.83.31.63
+  - 10.83.31.64
+  - 10.83.31.65
+cvp_ingestauth_key: blah
```

Comment:
Please set it as variable (< CloudVision Ingest Authentication key >) instead of blah, unless if that is the actual value.

Reply:
Thanks, should be fixed with commit cd042104